Routing/RestRouteCollection.php

```diff
@@ -76,10 +76,17 @@ public function setDefaultFormat($format)
      */
     public function all()
     {
+        $regex = <<<REGEX
```

Comment:
I'm not a huge fan of this annotation :x
Maybe concatenating multiple strings would be better?

Anyway that's not a big point, @xabbuh what do you use in symfony for the long regexps?